packages/calendar/src/utils/stateless/events/event-styles.ts

```diff
@@ -10,6 +13,23 @@ export const getEventHeight = (
   dayBoundaries: DayBoundariesInternal,
   pointsPerDay: number
 ) => {
+  if (start === end) {
+    return (
+      timePointToPercentage(
+        pointsPerDay,
+        dayBoundaries,
+        timePointsFromString(
+          timeFromDateTime(addTimePointsToDateTime(end, 100))
```

Comment:
100 equals 1 hours here. Not saying it's wrong, but I'm curious why you think 1 hour is the correct way to display a one-minute event. I personally would have picked the lowest possible number here to be able to display any info at all, like the event title.

Google calendar seems to do something in between our both approaches: a half-hour in this case. What do you think?

Reply:
To be honest I didn't have an opinion on how to show it on other type of calendar than monthly. Google Calendar approach seems like a good compromise between showing it big enough but not too big tho.